pkg/authenticator/authenticator.go

```diff
@@ -314,3 +318,25 @@ func decodeFromPEM(PEMBlock []byte, publicCert *x509.Certificate, privateKey cry
 
 	return decodedPEM, nil
 }
+
+func getInjectClientCertError(path string) string {
```

Comment:
nit: Not a blocker but this should hint that we are removing the file too probably. Maybe `consumeCertInjectionError`?